Specify AdaptiveAvgPool-8 and AdaptiveMaxPool-8 (#5773)
openvinotoolkit/openvino: docs/ops/opset8.md

# Operation Set `opset8` Specification {#openvino_docs_ops_opset8}

This specification document describes the `opset8` operation set supported in OpenVINO™.

> Support for each particular operation from the list below depends on the capabilities available in an inference plugin

Reviewer: ```suggestion
Support for each particular operation from the list below depends on the capabilities of an inference plugin
```

and may vary among different hardware platforms and devices. Examples of operation instances are provided as IR V10 xml
snippets. Such IR is generated by the Model Optimizer. The semantics match corresponding nGraph operation classes

> declared in `namespace opset7`.

Reviewer 2: opset7 -> opset8
